pkg/initcontainer/initcontainer.go

```diff
@@ -0,0 +1,64 @@
+package initcontainer
```

Comment:
I'm questioning the name of this package. An "initContainer" is a Kubernetes pod construct and I think this is confusing me already. I think `nodeinit` would be a more prescriptive name.